### PR TITLE
Make image order editable on detail pages - changed in CXGN::Image backend

### PR DESCRIPTION
**Pull request by lukasmueller**

### CODE REVIEW

File: lib/CXGN/Image.pm [hunk 1]
```diff
@@ -190,7 +191,7 @@ sub store {
                             obsolete,
                             modified_date,
                             md5sum)
-                     VALUES (?, ?, ?, ?, ?, ?, now(), ?)";
+                     VALUES (?, ?, ?, ?, ?, ?, ?, now(), ?)";
```
Comment (nmenda) [inline lib/CXGN/Image.pm]: why is there another param here?

**Comment by lukasmueller** [comment]
We should switch the stock page over the CXGN::Stock, but currently it is still based on CXGN::Chado::Stock. 

The additional functions also need to be added to CXGN::Stock.

Do you want to migrate the page to CXGN::Stock?

cheers
Lukas

**Comment by nmenda** [comment]
yes, I'll switch the stock page to use CXGN::Stock

What about line 194 in CXGN::Image?
https://github.com/solgenomics/cxgn-corelibs/pull/20/files/4916fe0a804e8eac5e2ce713bcb88ff4d1cd96d5#diff-f8c61e2579fa9c2673e5205828427573

**Comment by lukasmueller** [comment]
I think there is a ? too many... could you remove?

**Pull request merged**